go.mod

```diff
@@ -308,25 +311,25 @@ require (
 	github.com/russross/blackfriday v1.6.0 // indirect
 	github.com/russross/blackfriday/v2 v2.1.0 // indirect
 	github.com/sassoftware/relic v0.0.0-20210427151427-dfb082b79b74 // indirect
-	github.com/secure-systems-lab/go-securesystemslib v0.3.1 // indirect
+	github.com/secure-systems-lab/go-securesystemslib v0.4.0 // indirect
 	github.com/shibumi/go-pathspec v1.3.0 // indirect
 	github.com/shopspring/decimal v1.2.0 // indirect
-	github.com/sigstore/rekor v0.4.1-0.20220114213500-23f583409af3 // indirect
+	github.com/sigstore/rekor v0.12.1-0.20220915152154-4bb6f441c1b2 // indirect
```

Comment:
This version of rekor uses go-tuf v0.5.0, which has the fix for https://github.com/theupdateframework/go-tuf/security/advisories/GHSA-66x3-6cw3-v5gj

https://github.com/sigstore/rekor/blob/4bb6f441c1b27ccc7e625c721c7d3203acc7b313/go.mod